src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx

```diff
@@ -105,6 +139,11 @@ function OrderSubmitStatusAlert({
         }
     }
 
+    const reDeployService = () => {
+        stopWatch.reset();
```

Comment:
the timer is still set to 0 and does not start after clicking the "retry request" button.